path_rotate_root.go

```diff
@@ -0,0 +1,149 @@
+package azureauth
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/go-multierror"
+	//"github.com/hashicorp/go-uuid"
```

Comment:
Any reason to not use `github.com/hashicorp/go-uuid`?

Reply:
Nope! Nice catch 😄 Updated in latest commit